README.md

```diff
@@ -1,2 +1,180 @@
+
+![modify source code](doc/source/images/modify-source-code.png)
+
+#### Option 2 -  Use lale library for semi automated data science
+
+By using the *hyperopt* optimizer provided by the *lale* framework, we see how an estimator modified as a graph can be trained and tested.
+
+![visualize -2](doc/source/images/visualize-2.png)
+
+Since the framework is compatible with scikit-learn the output pipeline can be exported as a scikit-learn model using the `export_to_sklearn_pipeline` method.
+
+![export sklearn](doc/source/images/export-sklearn.png)
+
+## 4. Deploy and score as webservice using WML instance
+
+This section of the notebook contains code that deploys the pipeline model as a web service using Watson Machine Learning. This section requires users to enter credentials to be able to identify the right WML instance and deployment space. The two credentials are `api_key` and `target_space_id`
+
+### api_key
+
+With Watson Machine Learning V4, the user no longer is needed to provide the username and password information to be able to access the WML instance. Instead, the user will need to generate an *api key*  through the cloud account and paste it in the cell as shown in the cell below. The instructions to acquire the *cloud api key* is described in the markdown section of the screenshot shown below.
+
+![wml connection](doc/source/images/wml_connection_autoai.png)
+
+### target_space_id
+
+>*To create a deployment space and get the target_space_id*:
+![wml deployment space](doc/source/images/wml-deployment-space.gif)
+>
+>1. Click on the hamburger menu on the top-left corner of the Watson Studio home page.
+>1. Click on `Deployment Spaces` from the list and select `View all spaces`
+>1. Click `New deployment space`, select `Create an empty space` option.
+>1. Provide a name, select a machine learning service that was previously created and click `Create`
+>1. Click `View new space` and switch to the `Settings` tab and copy the `space id`
+
+Acquire the *target_space_id* as shown in the steps above and paste within the create deployment section. The Watson Machine Learning API uses the `wml_credentials` and the `target_space_id` to deploy the machine learning model as a webservice.
+
+![create deployment](doc/source/images/create_deployment_autoai.png)
+
+Once the cells are executed, the model is promoted to the deployment space and is now available as a web service and can be verified from within the UI as shown below.
+
+![model deployed](doc/source/images/wml_model_deployed.png)
+
+### Score webservice
+
+Scoring the webservice can either be done from the UI by switching to the `test` tab shown in the screenshot above. Alternatively, the *score()* method from the WML API can be be used  to submit a sample test payload. The results are returned as shown in the screenshot below. 
+
+![score webservice](doc/source/images/score_webservice.png)
+
+# Related Links
```

Comment:
here also I put these on the web page instead since it was removed from our readme template, but if you really want a link here keep it.  I'd prefer you don't use an h1 though.

Reply:
makes sense. Will remove this one too.